sdks/python/apache_beam/examples/california_housing_clustering.py

```diff
@@ -40,17 +74,30 @@ def run(
   if not test_pipeline:
     pipeline = beam.Pipeline(options=pipeline_options)
 
-  data = (pipeline | read_csv(known_args.input))
+  data = pipeline | read_csv(known_args.input)
 
   features = ['longitude', 'latitude', 'median_income']
 
   housing_features = to_pcollection(data[features])
 
+  # 1. Calculate clustering centers and save model to persistent storage
   _ = (
       housing_features
       | beam.Map(lambda record: list(record))
-      | OnlineClustering(
-          OnlineKMeans, n_clusters=6, batch_size=256, cluster_args={})
+      | "Train clustering model" >> OnlineClustering(
+          OnlineKMeans,
+          n_clusters=6,
+          batch_size=256,
+          cluster_args={},
+          checkpoints_path='/tmp/checkpoints'))
+
+  # 2. Calculate labels for all records in the dataset
+  # using the trained clustering model
+  _ = (
```

Comment:
I don't think having 2 totally separate transforms is what we want. This transform should still be dependent on the first transform, and probably shouldn't use RunInference (since RunInference won't reload the model over time). Otherwise, this likely either won't work at all (and will throw when loading the model), or it will only ever use the cluster generated by the first few datapoints (and won't update over time)